mavlink/qgroundcontrol
# Change default orbit yaw behaviour

Change of default orbit yaw behaviour.
-----------
Associated PR: 

- [x] https://github.com/PX4/PX4-Autopilot/pull/23358
- [x] https://github.com/mavlink/mavlink/pull/2132

Description
-----------
when a vehicle type, for which the orientation is relevant, performs an orbit maneuver, e.g a Tricopter, the orientation during the orbit had to be hardcoded. With this PR i change [ORBIT_YAW_BEHAVIOUR](https://mavlink.io/en/messages/common.html#ORBIT_YAW_BEHAVIOUR) from [ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER](https://mavlink.io/en/messages/common.html#ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER) to  [ORBIT_YAW_BEHAVIOUR_UNCHANGED](https://mavlink.io/en/messages/common.html#ORBIT_YAW_BEHAVIOUR_UNCHANGED) .
if the user uses a PX4 version where the mentioned PR is not active, it remains ORBIT_YAW_BEHAVIOUR_HOLD_FRONT_TO_CIRCLE_CENTER as this is what is set in the constructor of MAV_CMD_DO_ORBIT. so backwards compatibility is given. if The user used a PX4 version with the mentioned PR, it allows the user to set the orbit yaw behaviour based on a vehicle parameter.


Test Steps
-----------
Tested in SITL


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
